src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/NotifyClaimCallbackHandler.java

```diff
@@ -190,6 +190,11 @@ private CallbackResponse submitClaim(CallbackParams callbackParams) {
             claimDetailsNotificationDeadline = getDeadline(claimNotificationDate);
         }
 
+        if (claimDetailsNotificationDeadline.isAfter(caseData.getClaimNotificationDeadline())
+            || claimDetailsNotificationDeadline.isEqual(caseData.getClaimNotificationDeadline())) {
+            claimDetailsNotificationDeadline = caseData.getClaimNotificationDeadline();
+        }
```

Comment:
why do you need an if block here, why it can't be simply -
claimDetailsNotificationDeadline = caseData.getClaimNotificationDeadline();
All we need is claimDetailsNotificationDeadline to be same as claimNotificationDeadline in any case right?

Reply:
No Akriti we need to check if we are crossing the 4 month then only we want claimDetailsNotificationDeadline = claimNotificationDeadline  otherwise we will add 14 days and that can be different if claimDetailsNotificationDeadline is much less than 4 month end point